src/keys/mod.rs

```diff
@@ -371,45 +371,40 @@ where
 /// implements it, the returned [`GeneratedKey`] will also implement it.
 pub trait GeneratableKey<Ctx: ScriptContext>: Sized {
     /// Lenght in bytes of the required entropy
```

Comment:
nit: this doc comment should probably be updated

Reply:
fixed